optimism/src/lookup.rs

```diff
@@ -39,7 +39,7 @@ pub struct Lookup<Fp> {
     pub mode: LookupMode,
     /// The number of times that this lookup value should be added to / subtracted from the lookup accumulator.    pub magnitude_contribution: Fp,
     pub magnitude: Fp,
-    pub table_id: LookupTable,
+    pub table_id: LookupTables,
```

Comment:
Maybe it's better to rename `table_id` into `table_type`? I assume you can extract the id from the enum, but it's looking confusing as it is.